TACC/tup-ui
# Task/tup 373 project summary listing

## Overview:
Created one column table to eventually be able to map to separate systems table component for Projects page. 

## Related:

- [TUP-373](https://jira.tacc.utexas.edu/browse/TUP-373)

## Changes:

- Created new ProjectSummaryListing component. 
- Added new cells to ProjectCells. Added types to the existing ProjectRawSystem type in tup-hooks. 
- Added tests for ProjectSummaryListing component.

## Testing:

1. Go to [localhost:8000/dashboard](localhost:8000/dashboard)
2. Click the Projects tab from the sidebar
3. Make sure listing populate correctly. Compare with the existing TUP site [here.](https://portal.tacc.utexas.edu/projects-and-allocations)
4. Go back to dashboard and click the link above the Projects table
5. Make sure the results are the same as the other view. 

## UI:
<img width="658" alt="Screen Shot 2022-12-13 at 1 54 57 PM" src="https://user-images.githubusercontent.com/96220951/207431199-61e1b30b-1330-42a6-897c-867198d76e0f.png">

## Notes:
- Please send feedback! Tried to keep styling as dry as possible. I want to refactor eventually so that a hook gathers the allocation usage rather than having the calculations inside component. 

-There's some discrepancies in the numbers between existing TUP and what I'm getting back in the response for my usage specifically for the Advanced Computing Interfaces' Project TACC-ACI. 

 ** Is this because the old TUP is still factoring in Longhorn and the new TUP is not?
